eclass/qt5-build.eclass

```diff
@@ -316,6 +326,16 @@ qt5-build_pkg_postrm() {
 
 ######  Public helpers  ######
 
+# @FUNCTION: qt5_symlink_binary_to_path
+# @USAGE: <target binary name> [suffix]
+# @DESCRIPTION:
+# Symlink a given binary from QT5_BINDIR to QT5_PREFIX/bin, with optional suffix
+qt5_symlink_binary_to_path() {
+	[[ $# -ge 1 ]] || die "${FUNCNAME}() requires at least one argument"
+
+	dosym -r "${QT5_BINDIR#${EPREFIX}}"/${1} /usr/bin/${1}${2}
```

Comment:
Do we actually need the #$...? It figures out the shortest path anyway? Or what am I missing?